marketplace.xml

```diff
@@ -7594,4 +7594,39 @@ More on: https://anotherreeshu.wordpress.com/2015/01/13/special-character-remove
   </screenshots>
 </market_entry>
 
+  <market_entry>
+    <id>pdi-sequoiadb-plugin</id>
+    <type>Step</type>
+    <name>PDI SequoiaDB Plugin</name>
+    <category>
+      <name>Big Data</name>
+      <parent>
+        <name>PDI Plugins</name>
+      </parent>
+    </category>
+    <description>Pentaho data integration plugin for SequoiaDB</description>
+    <author>SequoiaDB</author>
+    <documentation_url>https://github.com/SequoiaDB/pentaho-sequoiadb-plugin/blob/master/README.markdown</documentation_url>
+    <versions>
+      <version>
+        <branch>Stable</branch>
+        <version>1.0.0</version>
+        <package_url>https://github.com/SequoiaDB/pentaho-sequoiadb-plugin</package_url>
+        <source_url>https://github.com/SequoiaDB/pentaho-sequoiadb-plugin</source_url>
+        <development_stage>
+          <lane>Customer</lane>
+          <phase>3</phase>
+        </development_stage>
+      </version>
+    </versions>
+    <license_name>Apache License 2.0</license_name>
+    <license_text>For more details see:
+      https://github.com/SequoiaDB/pentaho-sequoiadb-plugin/blob/master/LICENSE
+    </license_text>
+    <support_level>COMMUNITY_SUPPORTED</support_level>
+    <support_message>Supported by a SequoiaDB Enterprise Edition support contract.</support_message>
+    <support_organization>SequoiaDB</support_organization>
+    <support_url>http://bbs.sequoiadb.com/cn/forum.php?mod=forumdisplay&fid=2</support_url>
```

Comment:
The ampersand needs to be encoded (replace with &amp; or the whole thing in a CDATA tag)